packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -469,6 +469,32 @@ class WebViewController {
   }
 }
 
+/// Manages cookies pertaining to all [WebView]s.
+class CookieManager {
+  factory CookieManager() {
```

Comment:
This needs a Dartdoc, it should mention that it returns or created the singleton instance.

Reply:
nit: not sure if '--' is common in Flutter dartdocs (of if it is grammatically correct), I'd just say:
```
Returns the singleton cookie manager instance.

The instance is created the first time this factory is called.
```